src/ActionModal/components/DelegationModal.vue

```diff
@@ -399,6 +406,34 @@ export default {
       update(data) {
         return data.balance || { amount: 0 }
       }
+    },
+    totalStaked: {
+      query: gql`
+        query overview($networkId: String!, $address: String!) {
+          overview(networkId: $networkId, address: $address) {
+            liquidStake
+            totalStake
+          }
+        }
+      `,
+      skip() {
+        /* istanbul ignore next */
+        return (
+          (!this.address || !this.network) &&
+          // only needed for polkadot to determine if user needs to set an amount
+          this.network !== "polkadot-testnet"
+        )
+      },
+      variables() {
+        /* istanbul ignore next */
+        return {
+          networkId: this.network,
+          address: this.address
+        }
+      },
+      update({ overview: { totalStake, liquidStake } }) {
+        return totalStake - liquidStake
```

Comment:
We could still do this with two queries... Or maybe we don't remove `totalStake` after all in the API. It is convenient in this case